sympy/physics/control/tests/test_control_plots.py

```diff
@@ -149,7 +151,7 @@ def test_bode_data(sys):
 
 
 def check_point_accuracy(a, b):
-    return all(Abs(a_i - b_i) < 1e-12 for \
+    return all(((Abs(a_i - b_i) < 1e-12) or (Abs((a_i / b_i) - 1) < 1e-8)) for \
```

Comment:
I think it should be possible to replace this with `math.isclose`. If one only makes it possible to traverse arbitrary structure of numbers (tuples of tuples etc), is should be possible to use that for the above tests. Something like: